sources/packages/app/src/index.tsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { crafterConf } from '@craftercms/classes';
+// @ts-ignore
+import cookies from 'js-cookie';
```

Comment:
Isn't there a `yarn add @types/js-cookie -D` that fixes this?

Reply:
Added `@types/js-cookie`